src/main/java/creativei/vo/AddressDataVo.java

```diff
@@ -0,0 +1,44 @@
+package creativei.vo;
+
+import creativei.entity.State;
+
+import java.util.List;
+
+public class AddressDataVo {
```

Comment:
Rename to LocalityVo

Reply:
sir there is already a LocalityVo  which has name and id as its fields. AddressDataVo containing the data related to the address.

Reply:
Sir it contains fields like country and state too CityVo should be related to city only .I have already created the cityVo too which contains city name and its id.